cypress/e2e/API/api_EngageSphere.cy.js

```diff
@@ -0,0 +1,160 @@
+/// <reference types="cypress" />
+
+//DÚVIDAS!!!
+//to.be.at.greaterThan(9999).and.to.be.lessThan(50000)
+//Retorna que o customer Littel  tem 10.000 funcionários e também que ele possuí menos de 50.000
+
+describe('ENGAGE SPHERE API TESTING', () => {
+
+    const CUSTOMERS_API_URL = `${Cypress.env('API_URL')}`
+
+    it('SUCCESSFULLY RETRIEVES CUSTOMERS (E.G., CHECKS FOR THE 200 STATUS CODE)', () => {
+        cy.request('GET', CUSTOMERS_API_URL + '/customers')
+            .then(({ body, status }) => {
+                expect(status).to.be.eq(200);
+                expect(body.pageInfo.totalCustomers).to.be.is.not.null;
+
+            });
+    })
+    it('RETURNS THE CORRECT STRUCTURE OF THE RESPONSE OF THE FIRST PAGE OF CUSTOMERS(I.E., CUSTOMERS AND PAGEINFO PROPERTIES)', () => {
+        cy.request('GET', CUSTOMERS_API_URL + '/customers?page=1')
+            .then(({ body, status }) => {
+                expect(status, 'Status code').to.be.eq(200);
+
+                body.customers.forEach((customer) => {
+                    expect(customer, 'Customer id').to.have.property('id').that.is.a('number');
+                    expect(customer, 'Customer name').to.have.property('name').that.is.a('string');
+                    expect(customer, 'Number of employees').to.have.property('employees').that.is.a('number');
+
+                    // Verifique se contactInfo é null ou um objeto
+                    if (customer.contactInfo) {
+                        expect(customer.contactInfo).to.be.an('object');
+                        expect(customer.contactInfo, "Name for contact").to.have.property('name')
+                        expect(customer.contactInfo, 'Email for contact').to.have.property('email').to.be.eq(`${customer.contactInfo.email}`)
```

Comment:
Aqui eu deixaria só o seguinte, visto que o **dado** (o email em si) não é importante para o teste.
O propósito do teste é verificar a **estrutura** retornada na resposta.

```suggestion
                        expect(customer.contactInfo, 'Email for contact').to.have.property('email').that.is.a('string')
```